crates/rover-client/src/headers.rs

```diff
@@ -0,0 +1,37 @@
+use crate::RoverClientError;
+use reqwest::header::{HeaderMap, HeaderValue};
+use std::env;
+
+const CONTENT_TYPE: &str = "application/json";
+const CLIENT_NAME: &str = "rover-client";
+
+/// Function for building a [HeaderMap] for making http requests.
+///
+/// Takes a single argument, "api_key"m and returns a [HeaderMap].
+pub fn build(api_key: &str) -> Result<HeaderMap, RoverClientError> {
+    let mut headers = HeaderMap::new();
+
+    let content_type = HeaderValue::from_str(CONTENT_TYPE).expect("is valid header");
```

Comment:
```suggestion
    let content_type = HeaderValue::from_str(CONTENT_TYPE)?;
```

this works because of the change i suggested to to the `RoverClientError::InvalidHeaderError` type